src/components/Overlay/OverlayToolbar.js

```diff
@@ -47,41 +52,31 @@ const styles = {
       flexBasis: '48px'
     }
   }),
-  confirm: css({
-    height: `${height}px`,
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
-    cursor: 'pointer',
-    border: 'none',
-    outline: 'none',
-    background: 'transparent',
-    ...sansSerifRegular16,
-    margin: '0 0 0 auto',
-    padding: '0 12px',
-    [mUp]: {
-      padding: '0 20px'
-    }
+  tag: css({
+    ...sansSerifRegular15,
+    display: 'inline-block',
+    padding: '3px 9px',
+    borderRadius: 3,
+    lineHeight: '21px',
+    height: 28,
+    color: '#fff',
+    pointerEvents: 'none',
+    position: 'absolute',
+    right: 60,
+    top: 9
   })
 }
 
-export const OverlayToolbar = ({ children }) => {
+const Tag = ({ text }) => {
```

Comment:
Is this being used? Maybe I'm missing something or an artefact from trying things out?

Reply:
precisely. i remove